resources/mei_bibliography.bib

```diff
@@ -587,6 +597,21 @@ @misc{Morent_2006
  note = {{"Digitale Medien und Musikedition". Kolloquium des Ausschusses f{\"u}r musikwissenschaftliche Editionen der Union der deutschen Akademien der Wissenschaften. Mainz, Akademie der Wissenschaften}}
 }
 
+@incollection{Muennich_2023,
+ abstract = {Looking at the first three volumes of the published output from the ‘On the Genealogy of Writing’ project, there seems to be a categorical difference between writing scenes in the age of manuscripts, typescripts, or in the ‘digital age’, as it is somewhat emphatically called there. I would like to take up this differentiation and, linked to reflections on writing theory, trace the categorial differences between various types of musical writing scenes. In doing so, I will limit myself to four different types: one handwritten, one printed, one in the form of music encoding formats and their rendering (MEI and Verovio), and one that I address here as ‘semantically operationalised’. In doing so, I first elaborate observations pertaining to writing theory and findings in relation to the historically transmitted material in order to then draw conclusions about the underlying writing scene and the transductive processes negotiated in it.},
+ author = {Münnich, Stefan},
+ title = {Handwritten -- Encoded -- Semantically Operationalised: Musical Writing Scenes in the Digital Realm},
+ pages = {371--391},
+ publisher = {{Brill | Fink}},
+ isbn = {9783770567140},
+ series = {Theorie der musikalischen Schrift},
```

Comment:
the volume seems to be missing, I would add that

Reply:
Good catch, thanks! Added the missing volume